docs/source/whatsnew.rst

```diff
@@ -134,8 +134,9 @@ Several enhancements in Fabric v2.0 make these new private data patterns possibl
   collections when deploying chaincode in Fabric v2.0.  Implicit
   organization-specific collections can be used without any upfront definition.
 
-To learn more about the new private data patterns, see the documentation for
-private data sharing, private data collection configuration, and implicit collections.
+To learn more about the new private data patterns, see :doc:`private-data/private-data`
+and :doc:`private-data-arch` for documentation about private data sharing, private
+data collection configuration, and implicit collections.
```

Comment:
Both links resolve to 'Private Data' which looks weird. Should we give the pages distinct names like 'Private Data Concepts' and 'Private Data Details'?
The first link has info about private data sharing.
The second link has sections for private data collection configuration and implicit collections. If we can't link directly to the section title, then I'd suggest just link to the page and not call out the sections here.